Good-Bids/goodbids
# [#] Skip Accessibility Checker

# Summary

- Removes accessibility checker step from onboarding wizard
- Adjusts UI of other steps to accomodate change
- Removes unused API endpoint

## Issues

- #831

## Testing Instructions

1. Create a new site
2. Visit site and start onboarding wizard
3. See there is no accessibility checker step
4. Finish onboarding
